moveit_planners/ompl/ompl_interface/src/constraints_library.cpp

```diff
@@ -270,11 +270,12 @@ void ompl_interface::ConstraintsLibrary::loadConstraintApproximations(const std:
   std::ifstream fin((path + "/manifest").c_str());
   if (!fin.good())
   {
-    logWarn("Manifest not found in folder '%s'. Not loading constraint approximations.", path.c_str());
+    ROS_WARN_NAMED("constraints_library", "Manifest not found in folder '%s'. Not loading constraint approximations.",
```

Comment:
This should be `ompl.constraints_library`.

Reply:
The package name already is `ompl_interface`. Adding `ompl.xyz` would be redundant.

Reply:
Agree.